gcloud/datastore/batch.py

```diff
@@ -21,7 +21,7 @@ class Local(object):
 
 from gcloud.datastore import _implicit_environ
 from gcloud.datastore import helpers
-from gcloud.datastore import datastore_v1_pb2 as datastore_pb
+from gcloud.datastore import _datastore_v1_pb2 as datastore_pb
```

Comment:
To clarify: I don't know if adding this module to the auto-doc'd page (like we do for other modules) is the best idea.

I think a longer discussion needs to happen to decide exactly what we should expose from the proto directly to users. Ideally, if they need lower-level access to something, they should be able to drop to the proto and do that... I assume we need to get some commitment from the people who author that proto about whether it will be backwards compatible. If so, I have no problem "proxying" the interface guarantee forward.